components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/SyncableLoginsStorage.kt

```diff
@@ -262,38 +258,22 @@ class SyncableLoginsStorage(
     @Throws(LoginsStorageException::class)
     override suspend fun getPotentialDupesIgnoringUsername(login: Login): List<Login> =
         withContext(coroutineContext) {
-            conn.getStorage().potentialDupesIgnoringUsername(login.toServerPassword())
+            conn.getStorage().potentialDupesIgnoringUsername(login.toNativeLogin())
                 .map { it.toLogin() }
         }
 
     override fun close() {
         coroutineContext.cancel()
         conn.close()
     }
-
```

Comment:
For context: we don't want to support "stand-alone syncs" in the future, and this was unused, so though it was a good time to get rid of it. (Sadly we can't remove it from the component yet as iOS needs it)